### ISSUE TITLE
Training and Test Curves on CIFAR-10 (VGG19 and ResNet50)

### ISSUE THREAD

**Issue by rsundar96** [rsundar96/curriculum-learning-acceleration#3]
Following are the training curves obtained when running the Hacochen Curriculum Learning code on CIFAR-10. Following hyper parameters were used to obtain the best possible result that I could obtain.

#### Details on code execution (VGG19):
- Learning Rate: Starts at 0.1 and gradually decreases till 0.001 over equal intervals
- Batch Size: 100 (*default*)
- Epochs: 300
- Optimiser: SGD (*default*)
- Loss Function: Categorical Cross Entropy (*default*)

<img width="724" alt="Screenshot 2020-07-31 at 09 15 33" src="https://user-images.githubusercontent.com/19652780/89015094-6d00e700-d30e-11ea-9ac1-4dafa315509e.png">

<img width="724" alt="Screenshot 2020-07-31 at 09 17 30" src="https://user-images.githubusercontent.com/19652780/89015284-b0f3ec00-d30e-11ea-8433-8fc92603bcf8.png">

<img width="724" alt="Screenshot 2020-07-31 at 09 19 20" src="https://user-images.githubusercontent.com/19652780/89015450-f3b5c400-d30e-11ea-98bb-9ee959177b55.png">

<img width="724" alt="Screenshot 2020-07-31 at 09 20 38" src="https://user-images.githubusercontent.com/19652780/89015561-23fd6280-d30f-11ea-9fe1-53f95aa31a17.png">

<img width="724" alt="Screenshot 2020-07-31 at 09 24 16" src="https://user-images.githubusercontent.com/19652780/89015843-a5ed8b80-d30f-11ea-9ef4-220cbfde0661.png">

<img width="724" alt="Screenshot 2020-07-31 at 09 22 04" src="https://user-images.githubusercontent.com/19652780/89015674-55762e00-d30f-11ea-8df6-af084f84e082.png">

**Final Training Accuracy** (Curriculum Learning): 91.0%
**Final Test Accuracy** (Curriculum Learning): 81.2%

**Final Training Accuracy** (Random Initialisation): 88.0%
**Final Test Accuracy** (Random Initialisation): 81.3%

#### Details on code execution (ResNet50):
- Learning Rate: Starts at 0.1 and gradually decreases till 0.001 over equal intervals
- Batch Size: 64
- Epochs: 300
- Optimiser: SGD (*default*)
- Loss Function: Categorical Cross Entropy (*default*)

<img width="719" alt="Screenshot 2020-07-31 at 09 37 37" src="https://user-images.githubusercontent.com/19652780/89016969-7f305480-d311-11ea-91da-a6ea5c8d81ff.png">

<img width="723" alt="Screenshot 2020-07-31 at 09 39 06" src="https://user-images.githubusercontent.com/19652780/89017117-b7d02e00-d311-11ea-87b6-72ac0a35f875.png">

<img width="723" alt="Screenshot 2020-07-31 at 09 44 38" src="https://user-images.githubusercontent.com/19652780/89017610-7e4bf280-d312-11ea-9c9b-4c7517ececff.png">

<img width="723" alt="Screenshot 2020-07-31 at 09 45 39" src="https://user-images.githubusercontent.com/19652780/89017697-a3d8fc00-d312-11ea-8677-167b15b711f7.png">

<img width="723" alt="Screenshot 2020-07-31 at 09 43 29" src="https://user-images.githubusercontent.com/19652780/89017518-52c90800-d312-11ea-8445-c980d23040b0.png">

**Final Training Accuracy** (Curriculum Learning): 93.75%
**Final Test Accuracy** (Curriculum Learning): 81.44%

**Final Training Accuracy** (Random Initialisation): 92.18%
**Final Test Accuracy** (Random Initialisation): 81.92%

**Comment by rsundar96** [comment]
Adding Curriculum vs. Random Test Loss curves for both VGG19 and ResNet50

<img width="723" alt="Screenshot 2020-07-31 at 10 02 19" src="https://user-images.githubusercontent.com/19652780/89019269-f5828600-d314-11ea-8bed-16ab25fbf811.png">

<img width="723" alt="Screenshot 2020-07-31 at 10 00 22" src="https://user-images.githubusercontent.com/19652780/89019136-c53ae780-d314-11ea-96cd-b20d54bca48a.png">
